### ISSUE TITLE
Problem exporting a wall with a cut void

### ISSUE THREAD

**Issue by manelmarib** [Autodesk/revit-ifc#508]
When exporting a wall with a cut part made by a void, we experienced the following:
![image](https://user-images.githubusercontent.com/112946224/188622866-dc264da0-142f-4b37-9db2-fdd48d49bcff.png)

Exporter v.22.5.1.0
RVT 22.1.30.34

**Comment by manelmarib** [comment]
[REV PARA IFC.zip](https://github.com/Autodesk/revit-ifc/files/9497414/REV.PARA.IFC.zip)

**Comment by manelmarib** [comment]
We came back to the 22.4.2.0 so we could solve this, but we need to change to the newest version of the exported to solve other issues... so is it possible to check this on new releases?

**Comment by AngelVelezSosa** [comment]
Yes absolutely.  We will add this case to our regression testing so that there is no regression once it is fixed.

**Comment by manelmarib** [comment]
any update on this? @AngelVelezSosa

**Comment by SvenAmiet** [comment]
Hello, I've got the same problem here. any news on this?

**Comment by manelmarib** [comment]
any news on this? @AngelVelezSosa

**Comment by manelmarib** [comment]
any news on this? @AngelVelezSosa @JOuellette-Autodesk

**Comment by ArtemHnatko** [comment]
The issue is not reproducible in Revit 2023 through Revit 2026. Walls with void cutouts are exported correctly to IFC without geometry errors when using the latest IFC Exporter versions.

**Testing environment:**
- Autodesk Revit 2023 (IFC 23.4.1.0)
- Autodesk Revit 2024 (IFC 24.3.20.0)
- Autodesk Revit 2025 (IFC 25.4.0.27)
- Autodesk Revit 2026 (IFC 26.1.0.23)

![Image](https://github.com/user-attachments/assets/bf9708c7-ae7a-443f-8615-198a0dec9f98)
